src/js/disability-benefits/526EZ/tests/config/evidenceType.unit.spec.jsx

```diff
@@ -39,12 +39,32 @@ describe('Disability benefits 526EZ evidence type', () => {
         uiSchema={uiSchema}/>
     );
 
-    selectCheckbox(form, 'root_view:vaMedicalRecords');
-    selectCheckbox(form, 'root_view:privateMedicalRecords');
-    selectCheckbox(form, 'root_view:otherEvidence');
+    selectCheckbox(form, 'root_view:selectableEvidenceTypes_view:vaMedicalRecords', true);
```

Comment:
Not sure if this ever worked right, because `selectCheckbox` doesn't set a default value for the third argument. 🤷‍♂️

Reply:
Was this breaking when you were running it locally?

Reply:
It was, after I made changes to the schema.